src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -1030,16 +1035,16 @@ void ScriptEngine::RemoveNetworkPlugins()
     }
 }
 
-GameActions::Result ScriptEngine::QueryOrExecuteCustomGameAction(std::string_view id, std::string_view args, bool isExecute)
+GameActions::Result ScriptEngine::QueryOrExecuteCustomGameAction(const CustomAction* customAction, bool isExecute)
```

Comment:
If `customAction` should not never be `nullptr`, use a reference.

```suggestion
GameActions::Result ScriptEngine::QueryOrExecuteCustomGameAction(const CustomAction& customAction, bool isExecute)
```